core/v0/api/core.yaml

```diff
@@ -1400,7 +1400,7 @@ components:
           type: string
           description: 'Area code. This can be Pincode, ZIP code or any equivalent'
     Agent:
-      description: Describes an order executor
+      description: Describes the direct performer or driver or executor of the fulfillment of an order. In the healthcare domain it can be a doctor while in the mobility domain it can be a taxi driver.<br>This has properties such as name, image, dob, gender, cred, tags, phone, email and rateable<br>This schema is used in the agent property of the Fulfillment schema object<br>This is used in the following situations.<br><ul><li>This is typically used in the discovery stage when the BPP wants to convey details of the person who will be fulfilling the order so that the customer can make an informed decision about the product or service they need to purchase</li><li>This is typically used in the fulfillment stage when the BPP assigns an agent to fulfill the order post confirmation of the same</li></ul><br>This is used to allow the BPP to send the details of the person fulfilling the order to the end user.
```

Comment:
We should really change the name from agent to person (calling doctor etc an agent is not appropriate). Whether that person acts as an agent of some entity is contextual (e.g. in the case of delivery). But at schema level this should ideally be named "Person" who can take any role (driver, delivery agent, doctor, etc) based on context.

Reply:
Hi @pramodkvarma I have applied the "Unification over Standardization" design principle and have added `Person` as an duplicate attribute in `Fulfillment` schema. As a lot of NPs have already implemented `Agent` it would be difficult to deprecate it soon. Can you suggest a deprecation date after which we can remove `Agent` from the spec.

Reply:
My suggestion is to create an alias for `agent` with for more appropriate term like `person` and then move on without necessarily deprecating it.